src/overlays/actors/ovl_Obj_Kzsaku/z_obj_kzsaku.c

```diff
@@ -28,27 +34,99 @@ const ActorInit Obj_Kzsaku_InitVars = {
     (ActorFunc)ObjKzsaku_Draw,
 };
 
-#endif
-
-extern UNK_TYPE D_06000040;
-extern UNK_TYPE D_06001118;
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/ObjKzsaku_Init.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/ObjKzsaku_Destroy.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/func_80C08B60.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/func_80C08B7C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/func_80C08BBC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/func_80C08BD0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/func_80C08C84.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/func_80C08CB0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/ObjKzsaku_Update.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Obj_Kzsaku/ObjKzsaku_Draw.s")
+extern Gfx D_06000040[];
+extern CollisionHeader D_06001118;
+
+void ObjKzsaku_Init(Actor* thisx, GlobalContext* globalCtx) {
+    s32 pad;
+    ObjKzsaku* this = THIS;
+    CollisionHeader* col = NULL;
+
+    Actor_SetScale(&this->dyna.actor, 1.0f);
+    DynaPolyActor_Init(&this->dyna, 1);
+    CollisionHeader_GetVirtual(&D_06001118, &col);
```

Comment:
With objects now in, you should be able to include the relevant object header and use the correct symbol.